doc/gameloop.uml

```diff
@@ -0,0 +1,29 @@
+@startuml
+title Game loop
+
+start
+
+:Generate board;
+:Remove matches from board;
+:Draw board;
```

Comment:
It seems like board is being drawn only once, if I'm not wrong.

Reply:
No, the board is of course drawn every frame, but I specified draw board at the top to clarify that the board shouldn't be drawn until it's ready. I'll change it to start drawing board then.

Reply:
I think I just got confused, there is no arrow pointing to the start of the game loop.